src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -487,6 +512,17 @@ export class HaAutomationEditor extends KeyboardShortcutMixin(LitElement) {
         sub(this._config)
       );
     }
+
+    if (changedProps.has("entityRegistry")) {
+      const entry = this._getAutomationEntity(
+        this.entityRegistry,
+        this._entityId
+      );
+      this._entityRegistryUpdate = {
```

Comment:
Wouldn't this overwrite the changes made by the user when the entity is created in the registry?